funcx_endpoint/funcx_endpoint/executors/high_throughput/interchange.py

```diff
@@ -110,6 +110,7 @@ def __init__(
         provider=None,
         max_workers_per_node=None,
         mem_per_worker=None,
+        available_accelerators=(),
```

Comment:
[] is an empty list, not ()

Reply:
I'd like to keep the use of a tuple for the default argument, as the empty list (`[]`) is mutable and could cause weird errors. Would you accept changing the documentation to "sequence" or "collection" of strings?

Reply:
From a typed python perspective, `Sequence` is the right term here, yeah - a thing you can iterate and index into, without it being mutable.

Reply:
can change the in-code type annotations to `Sequence[str]` which accepts `()` as an empty sequence of strings.

Reply:
I just touched my own option, and left the others alone.